easy-ui-react/src/Banner/Banner.module.scss

```diff
@@ -0,0 +1,67 @@
+@use "../styles/common" as *;
+
+.variantSuccess {
+  @include component-token(
+    "banner",
+    "color",
+    theme-token("color.text.success.inverse")
+  );
+  @include component-token(
+    "banner",
+    "background_color",
+    theme-token("color.background.success")
+  );
+}
+
+.variantPrimary,
+.variantNeutral {
+  @include component-token(
+    "banner",
+    "color",
+    theme-token("color.text.inverse")
+  );
+}
+
+.variantPrimary {
+  @include component-token(
+    "banner",
+    "background_color",
+    theme-token("color.background.primary")
+  );
+}
+
+.variantNeutral {
+  @include component-token(
+    "banner",
+    "background_color",
+    theme-token("color.background.neutral")
+  );
+}
+
+.Banner {
+  @include component-token(
+    "banner",
+    "border_radius",
+    design-token("shape.border_radius.base")
+  );
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  border-radius: 0;
```

Comment:
**Question:** Is this necessary? I think since we specify it for >tablet on line 61 this can be deleted.
```suggestion
```

Reply:
from what i can tell on the design spec, it seems like on mobile the border radius is 0 and takes up the full width, but will check in with design about this to be cerain